dist/backdraft.js

```diff
@@ -826,11 +826,30 @@ _.extend(Plugin.factory, {
       <table cellpadding="0" cellspacing="0" border="0" class="table table-striped table-bordered"></table>\
```

Comment:
You can ignore this file in the context of this pull request, it's a `grunt build` of the other files in the repository.